src/styles/navigation_section.css

```diff
@@ -63,18 +63,19 @@
   display: none;
 }
 
-[role=banner] .coz-nav-pop--apps .coz-nav-group {
+/* coz-nav-pop--apps label is hashed coz-nav-pop--1612761714 */
+[role=banner] .coz-nav-pop--1612761714 .coz-nav-group {
```

Comment:
Yep, hash will always be the same, it's a checksum from the label. I don't want to use the label itself as it can be inconsistent (spaces, diacritics, etc). I first used the `slug` module bu it comes with a ~1.5Mb dictionnary, bundled into the lib, so it's clearly a no-go :tongue:

I fallback to a simple checksum (cf https://github.com/cozy/cozy-bar/pull/13/files#diff-8f646533758ab8608cfd8140b4c1297cR79) based on the label. It's lightweight and as consistent as we need (very similar to CSS modules, in fact)